docs/source/index.rst

```diff
@@ -0,0 +1,84 @@
+.. Cleanlab documentation master file, created by
+   sphinx-quickstart on Mon Jan 10 07:17:00 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Introduction
+============
+
+**Cleanlab automatically finds and fixes errors in your ML datasets.**
+
+| This reduces manual work needed to fix data issues and helps train reliable ML models on partially mislabeled datasets. Cleanlab has already found thousands of `label errors <https://labelerrors.com>`_ in ImageNet, MNIST, and other popular ML benchmarking datasets, so let's get started with yours!
+
+.. note::
+   Cleanlab currently only supports classification tasks.
```

Comment:
remove this, it seems more appropriate for some sort of FAQ section

Reply:
done!